### ISSUE TITLE
unable to open acvm in my MBA m1

### ISSUE THREAD

**Issue by jsparrow08** [KhaosT/ACVM#37]
I am not able to open acvm in my Mac . 
<img width="995" alt="Screenshot 2021-07-19 at 2 01 24 AM" src="https://user-images.githubusercontent.com/77740824/126135618-6720c205-a7ae-4bb4-9cc7-46ccfc61164d.png">
 it shows that acvm will damage your computer and also showing that you do not have permission to open the appliation .

**Comment by KhaosT** [comment]
Yeah the developer account used to sign the build was removed by Apple. If you are looking for QEMU based VM solution, try https://github.com/utmapp/UTM
